index.html

```diff
@@ -1933,6 +1933,24 @@ <h3 id="Error Messages" style="left: -46px; position: relative"> Error Messages<
         longer error messages and helps keep the 100-character limit on lines as mentioned earlier in the
         style guide </p>
   </div>
+  <!-- Production Tests -->
+  <h3 id="Production Tests" style="left: -46px; position: relative"> Production Tests</h3>
+  <div class="stylebody">
+    <p> Production tests may be resource intensive and may take some time to run; these are categorised separately from other tests. Production tests do not run when the user enters ‘make test’ (being non-important and/or expensive in execution time). 
+<br>They should be denoted by adding ‘_prod’ at the end of the test name. For example: 
+</p>
+    
+
+    
+    <pre>TEST_F(DifferentiatorFunctionality, handlesZeroInput_prod){
```

Comment:
I'd change the example here to generalize the test fixture name and the test name since Differentiator wouldn't be a valid production test. Try something like TEST_F(PodSimulations,  SomePodSimulation_prod)